dev/com.ibm.ws.jca_fat_mbean/publish/servers/com.ibm.ws.jca.jdbc.mbean.fat.app.connPool/server.xml

```diff
@@ -0,0 +1,106 @@
+     </dataSource>		
+     	
+    <!-- Note that this datasource should only by used by the testExceptionInCreate test -->	
+     <dataSource id="exceptionInCreateDS" jndiName="jdbc/exceptionInCreateDS" jdbcDriverRef="FATJDBCDriver">		
+         <connectionManager maxPoolSize="2" connectionTimeout="0s"/>		
+     	<properties.derby.embedded databaseName="memory:exceptionInCreateDS" createDatabase="create" user="dbuser1" password="{xor}Oz0vKDtu" />		
+     </dataSource>
+     
+     <!-- Note that this datasource should only by used by the testMultiThread test -->		
+     <dataSource id="multiThreadDS" jndiName="jdbc/multiThreadDS" jdbcDriverRef="FATJDBCDriver">
+ 		<properties.derby.embedded databaseName="memory:multiThreadDS" createDatabase="create" user="dbuser1" password="{xor}Oz0vKDtu" />		      	<properties.derby.embedded databaseName="memory:ds1" createDatabase="create" user="dbuser1" password="{xor}Oz0vKDtu" />
+      </dataSource>
+    
+    <connectionFactory jndiName="eis/mak">
+      <connectionManager maxPoolSize="5" connectionTimeout="0"/>
+      <!-- Specify adapterName and propertyW to workaround precedence issues in full profile. -->
+      <properties.FAT1 />
+    </connectionFactory>
+
+    <jmsConnectionFactory id="cf1" jndiName="jms/cf1">
+      <connectionManager maxPoolSize="2" connectionTimeout="0"/>
+      <properties.FAT1 tableName="cf1tbl" userName="CF1USER"/>
+    </jmsConnectionFactory>
+    <jmsConnectionFactory id="cf6" jndiName="jms/cf6">
+      <connectionManager maxPoolSize="6" connectionTimeout="0"/>
+      <properties.FAT1 tableName="cf6tbl" userName="CF6USER"/>
+    </jmsConnectionFactory>
+    
+    <connectionManager id="externalConnectionManager" maxPoolSize="20"/>
+
+    <jmsQueue id="queue1" jndiName="jms/queue1">
+      <properties.FAT1 queueName="queue1"/>
+    </jmsQueue>
+    
+    <authData id="activation1auth" password="{xor}HhwLCW4PCBs=" user="ACTV1USER"/>
+
+    <jmsActivationSpec id="fvtapp/fvtweb/FVTMessageDrivenBean" authDataRef="activation1auth">
+      <properties.FAT1 destinationRef="queue1"/>
+    </jmsActivationSpec>
+
+    <jmsTopic id="topic1">
+      <properties.FAT1 topicName="topic1"/>
+    </jmsTopic>
+    
+    <jmsTopic id="topic2" jndiName="jms/topic2">
+      <properties.FAT1 topicName="topic2"/>
+    </jmsTopic>
+    
+    <jmsActivationSpec id="FVTMessageDrivenBeanBindingOverride">
+      <properties.FAT1 destinationRef="topic1" destinationType="javax.jms.Topic"/>
+    </jmsActivationSpec>
+    <resourceAdapter name="ra/rafat1" id="FAT1" location="${server.config.dir}/connectors/JCAFAT1.rar">
+      <classloader commonLibraryRef="DerbyLib"/>
+      <properties.FAT1 databaseName="jcafvtdb" createDatabase="true" password="{xor}DR4Nbg8IGw==" userName="RAR1USER"/>
+    </resourceAdapter>
+    
+    <!-- This is used by the fake RAR -->
+    <library id="DerbyLib">
+      <fileset dir="${server.config.dir}/derby" includes="derby.jar"/>
```

Comment:
FYI, when you remove Derby from git and instead use gradle to download it at build time this location will change to `${shared.resource.dir}/derby`

Reply:
done